src/Contracts/Rules/IRuleClosure.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Moves\Eloquent\Verifiable\Rules\Calendar\Contracts\Rules;
+
+use DateTimeInterface;
+use Moves\Eloquent\Verifiable\Contracts\IRule;
+use Moves\FowlerRecurringEvents\Contracts\ACTemporalExpression;
+
+/**
+ * Interface IRuleOpenClose
+ *
+ * Rule which verifies that an event takes place between a certain open and close time.
+ * Close time is currently limited to occur after open time on the same calendar date.
+ * That is, close time after midnight is not currently supported.
+ */
+interface IRuleClosure extends IRule
```

Comment:
This is a relatively minor complaint, but the naming here could be confusing for later programmers, as the word 'closure' is a loaded term in programming already. A synonymous word would be more comprehensible imo. IIRC, Moves refers to these as 'blocks', but even a slightly different wording such as `IRuleUnavailable` or similar would work as well

Reply:
That's a really good catch - I didn't think of that at all! I'll do a rename refactor on this and everything associated with it